### PR TITLE
Full CPU reproducibility for single-node federation

### PR DESCRIPTION
**Pull request by itrushkin**
- Modifies TinyImageNet PyTorch example to single envoy
- Adds non-federated case notebook to reproduce metrics

### CODE REVIEW

**Review by dmitryagapov — APPROVED** [review]

**Review by aleksandr-mokrov — APPROVED** [review]

File: openfl-tutorials/interactive_api/PyTorch_TinyImageNet/workspace/pytorch_tinyimagenet.ipynb [hunk 1]
```diff
@@ -296,13 +297,15 @@
...
     "    def __init__(self):\n",
+    "        torch.manual_seed(0)\n",
     "        super(Net, self).__init__()\n",
     "        self.model = torchvision.models.mobilenet_v2(pretrained=True)\n",
     "        self.model.requires_grad_(False)\n",
     "        self.model.classifier[1] = torch.nn.Linear(in_features=1280, \\\n",
     "                        out_features=200, bias=True)\n",
     "\n",
     "    def forward(self, x):\n",
+    "        torch.manual_seed(0)\n",
```
Comment (alexey-gruzdev) [inline openfl-tutorials/interactive_api/PyTorch_TinyImageNet/workspace/pytorch_tinyimagenet.ipynb]: I believe we discussed that this should be fixed, since fixing a random seed inside `forward()` leads to multiple seed fixes during each `forward()` call

File: openfl-tutorials/interactive_api/PyTorch_TinyImageNet/workspace/Non-Federated Case.ipynb [hunk 2]
```diff
@@ -0,0 +1,296 @@
+{
```
Comment (alexey-gruzdev) [inline openfl-tutorials/interactive_api/PyTorch_TinyImageNet/workspace/Non-Federated Case.ipynb]: Can we add a title and short description of what this notebook is designed for?

File: openfl-tutorials/interactive_api/PyTorch_TinyImageNet/workspace/Non-Federated Case.ipynb [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+    "    def __init__(self):\n",
+    "        torch.manual_seed(0)\n",
+    "        super(Net, self).__init__()\n",
+    "        self.model = torchvision.models.mobilenet_v2(pretrained=True)\n",
+    "        self.model.requires_grad_(False)\n",
+    "        self.model.classifier[1] = torch.nn.Linear(in_features=1280, \\\n",
+    "                        out_features=200, bias=True)\n",
+    "\n",
+    "    def forward(self, x):\n",
+    "        torch.manual_seed(0)\n",
```
Comment (alexey-gruzdev) [inline openfl-tutorials/interactive_api/PyTorch_TinyImageNet/workspace/Non-Federated Case.ipynb]: The same problem as described in FL case notebook.
